c10/core/DeviceType.h

```diff
@@ -31,6 +31,12 @@ enum class DeviceType : int16_t {
   ONLY_FOR_TEST = 20901, // This device type is only for test.
 };
 
+constexpr DeviceType kCPU = DeviceType::CPU;
```

Comment:
were you planning to remove the defines in Backend.h?

Reply:
right, forgot to save the change in Backend.h..